docs/articles/documentation/reference/test-api/fixture/clientscripts.md

```diff
@@ -32,3 +32,97 @@ fixture
 > Relative paths resolve against the test file location.
 
 You can use the [page](../../../guides/advanced-guides/inject-client-scripts.md#provide-scripts-for-specific-pages) option to specify pages into which scripts should be injected. If this option is missing, TestCafe injects scripts into all pages visited during the test.
+
```

Comment:
To editors: no text was added or modified in this file except for the headers. Other changes are technical.